src/main/scala/zio/redis/api/Transactions.scala

```diff
@@ -0,0 +1,13 @@
+package zio.redis.api
+
+import zio.redis.Command
+import zio.redis.Input.{ StringInput, UnitInput, Varargs }
+import zio.redis.Output.{ StreamOutput, UnitOutput }
+
+trait Transactions {
+  final val discard = Command("DISCARD", UnitInput, UnitOutput)
+  final val exec    = Command("EXEC", UnitInput, StreamOutput)
+  final val multi   = Command("MULTI", UnitInput, UnitOutput)
+  final val unwatch = Command("UNWATCH", UnitInput, UnitOutput)
+  final val watch   = Command("WATCH", Varargs(StringInput), UnitOutput)
```

Comment:
Should be non-empty list of strings as an input.

Reply:
Fixed